CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# Contributing to this repo
+
+## Overview
+There are two components to the [dbt Hub](https://hub.getdbt.com/):
+
+- **hub.getdbt.com:** Our package registry
+    - https://github.com/dbt-labs/hub.getdbt.com (this repo!)
+    - An indexed collection of JSON files
+    - A static site, served by those same JSON files, where users can discover available packages
+    - Hosted on Netlify
+- **Hubcap:** The way the Hub stays up to date!
+    - https://github.com/dbt-labs/hubcap
+    - A script that:
+        - Catches new package releases when they are available (via GitHub releases)
+        - Parses out metadata—crucially, this package's dependencies on *other* Hub packages
+        - Records each release in a predictable format (JSON file)
+        - Automatically opens a PR against [hub.getdbt.com](http://hub.getdbt.com) to add that JSON file to its indexed collection
+    - Hosted on Heroku
+
+## Running locally
+
+### Development
+
+See the [installing Ruby for Mac](#installing-ruby-for-mac) and [troubleshooting](#troubleshooting) sections below as-needed.
+
+Clone this repository and install submodules:
+
+```
+git submodule update --init
+```
+
+...then install dependencies with `bundler`:
+
+```bash
+bundle install
+```
+
+...and run with middleman:
+
+```bash
+bundle exec middleman serve --port 4567
+```
+
+### Use with `dbt deps` from dbt Core
+
+You can set the `DBT_PACKAGE_HUB_URL` environment variable to specify where `dbt deps` should look to resolve dependencies [listed in `packages.yml`](https://docs.getdbt.com/docs/building-a-dbt-project/package-management#how-do-i-add-a-package-to-my-project):
+```shell
+export DBT_PACKAGE_HUB_URL=http://localhost:4567/
+```
+
+Or if you have a Netflify preview created by a pull request:
+```
+export DBT_PACKAGE_HUB_URL=https://deploy-preview-1717--flamboyant-mcclintock-92ba2d.netlify.app/
+```
+
+Suppose your `packages.yml` in your dbt project contains the following content:
+```yml
+packages:
+  - package: tnalpgge/cowsay
+    version: 0.3.4
+```
+
+Then `dbt deps` will use the JSON served from the following locations:
+- `data/packages/tnalpgge/cowsay/index.json`
+- `data/packages/tnalpgge/cowsay/versions/0.3.4.json`
+
+
+### Installing Ruby for Mac
+
+[https://antran.app/2021/m1_mac_part2/](https://antran.app/2021/m1_mac_part2/)
```

Comment:
What is this link for? Do I only need to read it if I have an M1 Mac? Does it replace the instructions below if I'm on an M1? Is it where the below instructions came from?

Reply:
Committed an update to incorporate this feedback.